toml/src/main/resources/META-INF/toml-only.xml

```diff
@@ -27,8 +27,15 @@
                          displayName="Cyclic feature dependency"
                          enabledByDefault="true" level="ERROR"
                          implementationClass="org.rust.toml.inspections.CargoTomlCyclicFeatureInspection"/>
+        <localInspection groupName="Rust"
```

Comment:
```suggestion
        <localInspection language="TOML" groupName="Rust"
```
it's just missed, I suppose? See `CargoTomlCyclicFeatureInspection` declaration above